css/side_panel.css

```diff
@@ -1,6 +1,15 @@
 .sp-container {
   background-color: var(--dark-two);
-  min-width: 280px;
+  display: flex;
+  flex-direction: column;
+  font-size: 16px;
+  gap: 3px;
+  max-height: 529px;
```

Comment:
Where does this `max-height` come from? Should this be 100%  or a CSS variable if the max height is determined by the overarching viewport?

Reply:
If the property is 100%, it can cause the height of the parent container to increase when opening subAgent rows. Defining a css variables in `constants.css` seems like a good solution.